pkgs/applications/science/electronics/simulide/default.nix

```
@@ -0,0 +1,56 @@
+{ appimageTools, symlinkJoin, lib, fetchurl, makeDesktopItem }:
+
+let
+  pname = "simulide";
+  version = "0.4.15";
+  release = "SR10";
+
+  src = fetchurl {
+    url = "https://launchpad.net/${pname}/${version}/${version}-stable/+download/SimulIDE_${version}-${release}.AppImage";
+    sha256 = "25f85f5262b39c8b25da042f11a0fd517c5d2750e81e538ac15992659ebea03c";
```

```suggestion
    hash = "sha256-JfhfUmKznIsl2gQvEaD9UXxdJ1DoHlOKwVmSZZ6+oDw=";
```
Please use the hash in SRI format by doing the following:
```
> nix hash to-sri --type sha256 25f85f5262b39c8b25da042f11a0fd517c5d2750e81e538ac15992659ebea03c
sha256-JfhfUmKznIsl2gQvEaD9UXxdJ1DoHlOKwVmSZZ6+oDw=
```

---

Ok, I just changed the hash.